components/fieldgroups/profile/profile-healthcare-member.schema.json

```diff
@@ -197,6 +224,14 @@
                 }
               }
             },
+            "xdm:healthConcerns": {
+              "title": "Health Concerns",
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "description": "Array list of all health concerns"
+            },
```

Comment:
Reminder to update plan object. Need to move the following attributes from outside of plan object to beneath the plan object. Attributes are: 

primaryMemberID

beneficiaryRelationship

isDependent

billingAccountID